vm/src/stdlib/io.rs

```diff
@@ -643,6 +647,21 @@ mod fileio {
         flag as u32
     }
 
+    #[pyclass]
+    #[derive(Debug)]
+    struct FileIO {
+        fd: AtomicCell<i64>,
+        closefd: AtomicCell<bool>,
+    }
+
+    type FileIORef = PyRef<FileIO>;
+
+    impl PyValue for FileIO {
+        fn class(vm: &VirtualMachine) -> PyClassRef {
+            vm.class("_io", "FileIO")
+        }
+    }
```

Comment:
`impl Drop for FileIO { self.close() }`  ?

Reply:
I am thinking of `ty_del` to support manual calling `__del__`